lib/template-dependencies.ts

```diff
@@ -0,0 +1,47 @@
+const BASE_DEV_DEPENDENCIES = [
+  '@types/node',
+  '@vitest/coverage-istanbul',
+  'husky',
```

Comment:
husky is present as a direct dependency only in single repo apps. We use the same template also for apps inside a monorepo. In that case, husky is already added at the monorepo level.